src/hardware/sblaster.cpp

```diff
@@ -2112,7 +2114,7 @@ class SBLASTER final {
 
 	static constexpr auto blaster_env_name = "BLASTER";
 
-	std::unique_ptr<AutoexecObject> autoexec_line = {};
+	std::unique_ptr<AutoExecObject> autoexec_line = {};
```

Comment:
I find it weird that something that represents a line is called an "object". Why don't we simply call this `AutoExecLine` then?

Reply:
Object is gone, I'm now using plain `std::string`.